addons/folo/common/src/main/java/org/commonjava/indy/folo/data/FoloRecordCassandra.java

```diff
@@ -401,6 +414,7 @@ private void saveTrackedContentRecords(TrackedContent record) {
             String tracking_key = next.getString("tracking_key");
             trackingKeys.add(new TrackingKey(tracking_key));
         }
-        return trackingKeys;
+        //**/
```

Comment:
Can we remove this if there is no comments ?

Reply:
+1